README.md

```diff
@@ -0,0 +1,2 @@
+# jsonschema_doc_generator
+This sets of scripts aim to parse json schemas to translate them into restructuredText. Then, Sphinx will do the rest :) 
```

Comment:
For consistency with all other packages we are maintaining, also with Invenio peers, please generate the skeleton of this package using:
```
$ pip install cookiecutter
$ cookiecutter https://github.com/inveniosoftware/cookiecutter-invenio-module
```

This will fill in correct Copyright and all useful boilerplate.